docs/index.md

```diff
@@ -560,19 +559,7 @@ print(Settings().model_dump())
 ```
 
 To enable CLI parsing, we simply set the `cli_parse_args` flag to a valid value, which retains similar conotations as
-defined in `argparse`. Alternatively, we can also directly provide the args to parse at time of instantiation:
-
-```py
-from pydantic_settings import BaseSettings
-
-
-class Settings(BaseSettings):
-    this_foo: str
-
-
-print(Settings(_cli_parse_args=['--this_foo', 'is such a foo']).model_dump())
```

Comment:
Fixed suggestion of using private param `_cli_parse_args`. The "formal" equivalent is now:

```python
CliApp.run(Settings, cli_args=['--this_foo', 'is such a foo']).model_dump()
```